qiwis.py

```diff
@@ -449,6 +453,8 @@ class BaseApp(QObject):
     qiwiscallRequested = pyqtSignal(str)
     qiwiscallReturned = pyqtSignal(str, str)
 
+    _constants = namedtuple("emptynamespace", ())

```

Comment:
Is the type of `_constants` correct?
I think you intened that `_constants` is a namedtuple object, but currently it is a namedtuple class!

How about changing it as follows?
```python
_constants = namedtuple("emptynamespace", ())()
```

Reply:
Also please add the description of `constants` in the docstring.

Reply:
Nice review!

For the properties, I think we don't have to write the class docstring for it because we define them like methods.
We can write docstring in the property (method) definition.

Well, it would be good to make a rule "define properties before any other methods".